.github/pull_request_template.md

```diff
@@ -0,0 +1,18 @@
+## 1 Closes/fixes/resolves issue(s)?
+*TODO*
+
+## 2 What was added/changed/fixed?
+*TODO*
+
+## 3 Related issue(s)? (y/n)
```

Comment:
Just a suggestion, take what you like the most
```suggestion
## 3 Related issue(s)? [Optional]
```

Reply:
Yours better!